.github/workflows/npm-publish.yml

```diff
@@ -3,49 +3,50 @@ name: npm-publish
 on:
   workflow_dispatch:
     inputs:
-      package-name:
-        description: Package name
-        type: choice
-        default: "couper"
-        required: true
-        options:
-        - "couper"
-        - "@avenga/couper"
       package-version:
-        description: Package version
-        required: true
-        default: ${{ github.ref_name }}
+        description: Package version (empty = current tag)
+        default: 0.0.0
         type: string
-      token-name:
-        description: secret containing npm publish token
-        type: string
-        default: NPM_TOKEN
 jobs:
   npm:
     name: 'update npm package'
     runs-on: ubuntu-latest
+    env:
+      PACKAGE_NAME: "couper"
+      PACKAGE_VERSION: ${{ github.event.inputs.package-version || github.ref_name }}
+      # https://github.com/JS-DevTools/npm-publish/issues/15
+      INPUT_TOKEN: ''
     steps:
       - name: 'checkout'
         uses: actions/checkout@v2
       - name: 'prepare package'
         run: |
           cp LICENSE .npm
+          cp -r .npm .npm2
```

Comment:
As said above, matrix would be more readable

Reply:
Done!